ParabolInc/parabol
# feat: add integrations task for first-time users, use jsdom to attach the /integration link

# Description

Resolved #5943

> For first-time users, we add one more task that suggests to them to explore the available integrations.

## Demo

https://user-images.githubusercontent.com/4997466/172760932-a167e7cd-7b84-4892-a0a4-30886aa93604.mp4

## Testing scenarios

- create a new account
- go to the `on-borading` team
- verify these are two tasks
- clicks on the integration link
- successful go to the integration page

## Final checklist
